qiling/core_utils.py

```diff
@@ -46,9 +44,9 @@ def nprint(self, *args, **kw):
 
             # setup filter for logger
             # FIXME: only works for logging due to we might need runtime disable nprint, it should be a global filter not only syscall
-            # if self.strace != None and self.output == QL_OUTPUT.DEFAULT:
-            #     from .utils import Ql_filter
-            #     self.log_file_fd.addFilter(ql_filter(self.filter))
+            if self.filter != None and self.output == QL_OUTPUT.DEFAULT:
+                from .utils import Ql_filter
+                self.log_file_fd.addFilter(Ql_filter(self.filter))
```

Comment:
QL filter is a bad name, i admin i choose the name but we need a better one

Reply:
Now we need to figure this out. maybe just call it filter

Reply:
how about that we made Ql_filter as a setup function for now, maybe the name will be change after other filters is implemented in the future